core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -39,10 +39,35 @@
     try {
       return callable.call();
     } catch (AlluxioException e) {
-      LOG.debug("Internal Alluxio error when running rpc", e);
+      LOG.debug("Internal Error: {}", callable, e);
       throw e.toThrift();
     } catch (Exception e) {
-      LOG.error("Unexpected error running rpc", e);
+      LOG.error("Unexpected Error: {}", callable, e);
+      throw new UnexpectedAlluxioException(e).toThrift();
+    }
+  }
+
+  /**
+   * Calls the given {@link RpcCallable} and handles any exceptions thrown. This method also logs
+   * enter and exit when debug level logging is enabled.
+   *
+   * @param logger the logger to use for this call
+   * @param callable the callable to call
+   * @param <T> the return type of the callable
+   * @return the return value from calling the callable
+   * @throws AlluxioTException if the callable throws an Alluxio or runtime exception
+   */
+  public static <T> T call(Logger logger, RpcCallable<T> callable) throws AlluxioTException {
+    try {
+      logger.debug("Enter: {}", callable);
+      T ret = callable.call();
+      logger.debug("Exit: {}", callable);
```

Comment:
I have two concerns:

1) enabling automated log parsing requires a simple and consistent logging -- right now, our logging couples "Enter" with "Exit" or "Internal Error" or "I/O Error" or "Unexpected Error", which seems unnecessarily complicated

2) readability -- I find not using "Exit" upon encountering an exception confusing as the methods indeed exits ... someone not familiar with the above convention might look for "Exit" in the logs

How about using something along the following lines:

- `("Enter: {}", callable)` upon method entry
- `("Exit (OK): {}", callable)` upon clean method exit  
- `("Exit (Error): {}", callable, e)` upon unexpected method exit